dev/com.ibm.ws.wssecurity_fat.wsscxf.saml.1/build.gradle

```diff
@@ -33,7 +33,8 @@ dependencies {
                'net.sourceforge.htmlunit:neko-htmlunit:2.44.0',
                'xalan:xalan:2.7.2',
                project(':io.openliberty.org.apache.xercesImpl'),
-               'xml-apis:xml-apis:1.4.01'
+               'com.unboundid:unboundid-ldapsdk:5.1.0',
+               project(':com.ibm.ws.com.unboundid')
```

Comment:
You get the picture

Reply:
I was looking at just using project(':com.ibm.ws.com.unboundid') for all of the affected projects - I want to get the change delivered to get z/OS running again.
The same comment applies to all projects noted above.